components/empty-state/README.md

```diff
@@ -35,4 +35,23 @@ This is mainly used in an empty entity view.
 
 ## Empty state in use
 
+<<<<<<< HEAD
 <ba-ux-snippet name="empty-state-in-use"></ba-ux-snippet>
```

Comment:
I see that I totally botched a merge (or, rather, am not used to having to manually save each open file ;D), but where did the README content go? It seems to haven gotten sucked up by ux-snippet tags, but the content is no longer in the project anywhere?

Reply:
There are certain parts of the readme files, that are considered to be UX driven content. As the UX team should not be constrained to libraries release cycles or processes, a lot of contents in the readme have been replaced with `ba-ux-snippet` elements. They are being replaced during build times with more up-to-date content fetched from a CMS. 

The contents replacement with the `ba-ux-snippet` elements is correct.